research-software-directory/RSD-as-a-service
# 826 fix: order return items on search term

# Order organisation list by "best match" search term

Closes #826

Changes proposed in this pull request:
*   Search results for organisation at RSD and ROR are returned by "best match" of search value. Previously the list returned RSD items before ROR items independent of the "match quality"

How to test:
* `make start` to build app
* create new software 
* Search for organisation, the order of list items should be by best match on the value typed.
* create new project 
* Search for organisation, the order of list items should be by best match on the value typed.

PR Checklist:
